### ISSUE TITLE
Settings Add-ons lists not showing add-on version

### ISSUE THREAD

**Issue by crowbarz** [sabeechen/hassio-google-drive-backup#293]
The add-ons list in Settings (the one that appears when Partial Snapshots and/or Stop Addons are selected) aren't showing the add-on version any more:

```
Addons
[ ] Log Viewer (vundefined)
[ ] Node-RED (vundefined)
[ ] Home Assistant Google Drive Backup (vundefined)
...
```

I'm running 0.101.3 of your addon on top of HA Supervised on a self-managed Debian install, with supervisor 2020.10.0 and HA Core 0.116.4.

Incidentally, in case it's due to the same root cause, I also noticed that my supervisor REST add-on version checking sensor stopped working properly when the supervisor was upgraded to 2020.10.0. The `.addons[]` dict returned via http://supervisor/supervisor/info appears to have changed in the new supervisor version. I was checking `.version` and `.installed` for add-on version information, but looks like these were renamed `.version_latest` and `.version` respectively. This is the equivalent output from `ha su info` in the new supervisor (just before I upgraded your add-on :) :

```
$ ha su info --raw-json | jq '{"addons":[.data.addons[] | select(.update_available)]}'
{
  "addons": [
    {
      "name": "Hass.io Google Drive Backup",
      "slug": "cebe7a76_hassio_google_drive_backup",
      "description": "Automatically back up Hass.io snapshots to Google Drive",
      "state": "started",
      "version": "0.100.0",
      "version_latest": "0.101.3",
      "update_available": true,
      "repository": "cebe7a76",
      "icon": true,
      "logo": true
    }
  ]
}
```

**Comment by sabeechen** [comment]
I think your assessment of what went wrong is spot on.  They've been shuffling things around in the supervisor addon API recently, and unfortunately this isn't the first bug its caused me.

I'll include a fix in the next release

**Comment by sabeechen** [comment]
The just released version 0.102.0 should fix this too.  Please let me know or reopen this issue if you run into any trouble with the new version.
Also just a reminder, to make Home Assistant see the new version right away go to "Supervisor" > "Add-on Store" > Three dots in the top right corner > "Reload"
